.jenkins/pipelines/Azure/Linux/Jenkinsfile

```diff
@@ -269,6 +401,9 @@ try{
         "Host verification 1604 Release":         { ACCHostVerificationTest('16.04', 'Release') },
         "Host verification 1804 Release":         { ACCHostVerificationTest('18.04', 'Release') },
 
+        "Host verification package 1604 Release": { ACCHostVerificationPackageTest('16.04', 'Release') },
```

Comment:
Please add "RelWithDebInfo" to the test matrix, we release Linux packages in this mode.